lib/internal/util.js

```diff
@@ -257,6 +257,18 @@ function promisify(original) {
 
 promisify.custom = kCustomPromisifiedSymbol;
 
+// The build in join is relatively slow (currently v8 6.0)
+function join(output, separator) {
```

Comment:
This is weird, any idea why it's slower?

Reply:
That is a known issue in v8. There is a open ticket - I am a bit in a hurry but I can add the reference to it later on.

Reply:
@BridgeAR - no rush - I'm just curious - I approved the actual code changes - but I'd like to make sure this passes CITGM before release and I'm afraid that merging it would mean that it might be missed (since this is semver patch).

Reply:
https://bugs.chromium.org/p/v8/issues/detail?id=2897

It does not seem to be on the priority list though :smile: